app/script/view_model/bindings/CommonBindings.js

```diff
@@ -126,73 +126,63 @@ ko.virtualElements.allowedBindings.stopBinding = true;
 
-    if (lastHeight !== currentHeight) {
-      if (typeof callback === 'function') {
-        callback(currentHeight, lastHeight);
-      }
-      lastHeight = currentHeight;
-      const maxHeight = window.parseInt(getComputedStyle(element).maxHeight, 10);
+    let lastHeight = element.scrollHeight;
 
-      const isMaximumHeight = currentHeight >= maxHeight;
-      const newStyleOverflowY = isMaximumHeight ? 'scroll' : 'hidden';
-      element.style.overflowY = newStyleOverflowY;
+    const resizeTextarea = (textareaElement => {
+      textareaElement.style.height = 0;
+      const newStyleHeight = `${textareaElement.scrollHeight}px`;
+      textareaElement.style.height = newStyleHeight;
 
       if (syncElement) {
-        syncElement.style.overflowY = newStyleOverflowY;
+        syncElement.style.height = newStyleHeight;
       }
-      $(element).scroll();
-    }
-  };
 
-  return {
-    init(element, valueAccessor, allBindings, data, context) {
-      lastHeight = element.scrollHeight;
-      const params = ko.unwrap(valueAccessor()) || {};
-      triggerValue = params.triggerValue;
-      syncElement = document.querySelector(params.syncElement);
-      callback = params.callback;
-      throttledResizeTextarea = _.throttle(resizeTextarea, 100, {leading: !params.delayedResize});
-
-      if (triggerValue === undefined) {
-        return ko.applyBindingsToNode(
-          element,
-          {
-            event: {
-              focus: () => throttledResizeTextarea(element),
-              input: () => throttledResizeTextarea(element),
-            },
-          },
-          context
-        );
-      }
-    },
+      const currentHeight = textareaElement.clientHeight;
 
-    update(element, valueAccessor) {
-      // we need to consume the `valueAccessor` here in order
-      // for knockout to trigger this function (even if we actually don't need the value)
-      ko.unwrap(valueAccessor());
-      throttledResizeTextarea(element);
-    },
-  };
-})();
+      if (lastHeight !== currentHeight) {
+        if (typeof callback === 'function') {
+          callback(currentHeight, lastHeight);
+        }
+        lastHeight = currentHeight;
+        const maxHeight = window.parseInt(getComputedStyle(textareaElement).maxHeight, 10);
+
+        const isMaximumHeight = currentHeight >= maxHeight;
+        const newStyleOverflowY = isMaximumHeight ? 'scroll' : 'hidden';
+        textareaElement.style.overflowY = newStyleOverflowY;
+
+        if (syncElement) {
+          syncElement.style.overflowY = newStyleOverflowY;
+        }
+        $(textareaElement).scroll();
+      }
+    }).bind(null, element);
+    const throttledResizeTextarea = _.throttle(resizeTextarea, 100, {leading: !params.delayedResize});
+
+    throttledResizeTextarea();
+    if (triggerValue === undefined) {
+      return ko.applyBindingsToNode(
+        element,
+        {
+          event: {
+            focus: throttledResizeTextarea,
```

Comment:
Here I bound the `resizeTextarea` function to the element at creation time (no need to pass it for those callbacks anymore)